storage-proofs/src/settings.rs

```diff
@@ -23,6 +27,13 @@ impl Default for Settings {
             maximize_caching: false,
             merkle_tree_path: "/tmp/merkle-trees".into(),
             num_proving_threads: 1,
+            replicated_trees_dir: "".into(),
+            generate_merkle_trees_in_parallel: false,
+            // FIXME: Decide on a sensible default value, setting to `false`
+            //  at the moment for testing purposes only.
```

Comment:
Need to decide this.

Reply:
which option are you referring to with `disk-trees`?

Reply:
`generate_merkle_trees_in_parallel`

Reply:
I suppose should be `true` as it's the current behavior, but wanted to make sure.

Reply:
Misunderstood the original question I think, sorry, `disk-trees` is not an internal configuration option but the Rust feature that enables all the file-mapping related logic.

Reply:
right, you could set the default depending on the build flag

Reply:
You could put that logic into the config loading part, if you want

Reply:
in `Settings.new`? should that happen before or after the `try_into`?

Reply:
done